backend/src/main/java/ba/unsa/etf/si/bbqms/domain/Tenant.java

```diff
@@ -0,0 +1,121 @@
+package ba.unsa.etf.si.bbqms.domain;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import jakarta.persistence.*;
+
+@Entity
+@Table(name = "tenant")
+public class Tenant {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    private Long id;
+
+    @Column(name = "code")
+    private String code;
+
+    @Column(name = "name")
+    private String name;
+
+    @Column(name = "hq_address")
+    private String hqAddress;
+
+    @Column(name = "font")
+    private String font;
+
+    @Column(name = "welcome_message")
+    private String welcomeMessage;
+
+    @OneToOne
+    @JoinColumn(name = "logo_id", referencedColumnName = "id")
+    private TenantLogo logo;
+
+    public Tenant() {
+    }
+
+    public Tenant(final String code, final String name, final String hqAddress, final String font, final String welcomeMessage, final TenantLogo logo) {
```

Comment:
Formatting:
```
public Tenant(final String code,
              final String name,
              final String hqAddress,
              final String font, 
              final String welcomeMessage, 
              final TenantLogo logo) {
```

Generally, constructors with 3 or more attributes should have fields one below the other.

Reply:
Fixed formatting